--- z88dk/Makefile.orig	Wed Jan  8 20:23:27 2003
+++ z88dk/Makefile	Wed Jan  8 20:24:06 2003
@@ -86,7 +86,7 @@
 	cd libsrc ; $(MAKE) install
 
 install-libs:
-	mkdir -p $(prefix)/lib/z88dl/lib/clibs
+	mkdir -p $(prefix)/lib/z88dk/lib/clibs
 	cp -R lib/clibs/* $(prefix)/lib/z88dk/lib/clibs/
 	find $(prefix)/lib/z88dk/lib/clibs -type f | xargs chmod 644
 
@@ -101,7 +101,7 @@
 	cd src/sccz80 ; $(MAKE) PREFIX=$(prefix) install
 	cd src/z80asm ; $(MAKE) PREFIX=$(prefix) install
 	cd src/zcc ; $(MAKE) PREFIX=$(prefix) install
-	./config.sh $(prefix)/lib/z88dk $(DEFAULT)
+	./config.sh $(prefix)/lib/z88dk/ $(DEFAULT)
 	cp -R -p include $(prefix)/lib/z88dk
 	cp -R -p lib $(prefix)/lib/z88dk
 	find $(prefix)/lib/z88dk/include -name '*.h' | xargs chmod 644
