--- z88dk/Makefile.old	Wed Jan  8 20:00:30 2003
+++ z88dk/Makefile	Wed Jan  8 20:05:34 2003
@@ -92,22 +92,22 @@
 
 
 install:
-	mkdir -p -m 755 $(prefix)/bin $(prefix)/lib 
-	mkdir -p -m 755 $(prefix)/lib/z88dk  $(prefix)/lib/z88dk/lib
-	mkdir -p -m 755 $(prefix)/lib/z88dk/lib/clibs  $(prefix)/lib/z88dk/lib/config
-	cd src/appmake ; $(MAKE) PREFIX=$(prefix) install
-	cd src/copt ; $(MAKE) PREFIX=$(prefix) install
-	cd src/cpp ; $(MAKE) PREFIX=$(prefix) install
-	cd src/sccz80 ; $(MAKE) PREFIX=$(prefix) install
-	cd src/z80asm ; $(MAKE) PREFIX=$(prefix) install
-	cd src/zcc ; $(MAKE) PREFIX=$(prefix) install
+	mkdir -p -m 755 $(DESTDIR)/$(prefix)/bin $(DESTDIR)$(prefix)/lib 
+	mkdir -p -m 755 $(DESTDIR)/$(prefix)/lib/z88dk  $(DESTDIR)/$(prefix)/lib/z88dk/lib
+	mkdir -p -m 755 $(DESTDIR)/$(prefix)/lib/z88dk/lib/clibs  $(DESTDIR)/$(prefix)/lib/z88dk/lib/config
+	cd src/appmake ; $(MAKE) PREFIX=$(DESTDIR)/$(prefix) install
+	cd src/copt ; $(MAKE) PREFIX=$(DESTDIR)/$(prefix) install
+	cd src/cpp ; $(MAKE) PREFIX=$(DESTDIR)/$(prefix) install
+	cd src/sccz80 ; $(MAKE) PREFIX=$(DESTDIR)/$(prefix) install
+	cd src/z80asm ; $(MAKE) PREFIX=$(DESTDIR)/$(prefix) install
+	cd src/zcc ; $(MAKE) PREFIX=$(DESTDIR)/$(prefix) install
 	./config.sh $(prefix)/lib/z88dk/ $(DEFAULT)
-	cp -R -p include $(prefix)/lib/z88dk
-	cp -R -p lib $(prefix)/lib/z88dk
-	find $(prefix)/lib/z88dk/include -name '*.h' | xargs chmod 644
-	find $(prefix)/lib/z88dk/include -type d | xargs chmod 755
-	find $(prefix)/lib/z88dk/lib -type f | xargs chmod 644
-	find $(prefix)/lib/z88dk/lib -type d | xargs chmod 755
+	cp -R -p include $(DESTDIR)/$(prefix)/lib/z88dk
+	cp -R -p lib $(DESTDIR)/$(prefix)/lib/z88dk
+	find $(DESTDIR)/$(prefix)/lib/z88dk/include -name '*.h' | xargs chmod 644
+	find $(DESTDIR)/$(prefix)/lib/z88dk/include -type d | xargs chmod 755
+	find $(DESTDIR)/$(prefix)/lib/z88dk/lib -type f | xargs chmod 644
+	find $(DESTDIR)/$(prefix)/lib/z88dk/lib -type d | xargs chmod 755
 
 
 
